### ISSUE TITLE
Internal Server Error while downloading ISO with Rock 4C+ SBC overlay

### ISSUE THREAD

**Issue by DamiaPoquet** [siderolabs/image-factory#102]
The Talos Image Factory returns an Internal Server Error while downloading the artifacts for the Rock 4C+ overlay.

Steps to reproduce:

1. Go to https://factory.talos.dev/
2. Choose Talos v1.7.0-beta.0
3. Choose the rock4cplus (siderolabs/sbc-rockchip:v0.1.0-beta.0)
4. Submit

It will return the schematic ID `1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770 `

The arm64 kernel download URL successfully returns the kernel file, while the initramfs is returning Internal Server Error:
```
curl https://factory.talos.dev/image/1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770/v1.7.0-beta.0/initramfs-arm64.xz -v
*   Trying 4.157.170.221:443...
* Connected to factory.talos.dev (4.157.170.221) port 443
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://factory.talos.dev/image/1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770/v1.7.0-beta.0/initramfs-arm64.xz
> GET /image/1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770/v1.7.0-beta.0/initramfs-arm64.xz HTTP/2
> Host: factory.talos.dev
> User-Agent: curl/8.4.0
> Accept: */*
>
< HTTP/2 500
< date: Fri, 05 Apr 2024 18:42:05 GMT
< content-type: text/plain; charset=utf-8
< content-length: 22
< x-content-type-options: nosniff
< strict-transport-security: max-age=15724800; includeSubDomains
<
internal server error
* Connection #0 to host factory.talos.dev left intact
```

Other artifact downloads, like ISO or installer images are also failing to download.
```
docker pull factory.talos.dev/installer/1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770:v1.7.0-beta.0
Error response from daemon: received unexpected HTTP status: 500 Internal Server Error
```

**Comment by smira** [comment]
```
{"level":"info","ts":1712343675.5414426,"caller":"http/http.go:164","msg":"request","frontend":"http","method":"GET","path":"/image/1dcdb4d9d5793f5ba121cc685a9932787c7a60ad1f3492ee29d4fca80c2f0770/v1.7.0-beta.0/initramfs-arm64.xz","error":"error generating asset: failed to run overlay installer: fork/exec /tmp/imager99802320/overlay/installers/rock4cplus: no such file or directory, stdErr: "}
```

Looks like rock4cplus is broken?

**Comment by smira** [comment]
The actual issue is https://github.com/siderolabs/sbc-rockchip/issues/10, we'll take a look, thanks for reporting!

**Comment by smira** [comment]
This will be fixed with Talos v1.7.0-beta.1
